controllers/argocd/service.go

```diff
@@ -388,6 +400,11 @@ func (r *ReconcileArgoCD) reconcileRepoService(cr *argoproj.ArgoCD) error {
 		},
 	}
 
+	if cr.Spec.Repo.IsEnabled() && cr.Spec.Repo.IsRemote() {
```

Comment:
How is the behavior? Is there any dependency between `remote` and `enabled` field i.e  component should be `enabled` for `remote` to work?

Reply:
Conversation from slack chat:
> skips remote too, if enabled is set to false